CPS.md

```diff
@@ -529,7 +532,7 @@ Certificates may be administratively revoked by ISRG if it is determined that th
 
 ### 4.9.3 Procedure for revocation request
 
-Revocation requests may be made at any time via the ACME API.
+Revocation requests may be made at any time via the ACME API. Revocations with a reason code of `keyCompromise` will result in the affected key being blocked for future issuance and all currently valid certificates with that key will be revoked.
```

Comment:
A small tweak to clarify that the revocation request must be successful for it to have the desired effect:

```suggestion
Revocation requests may be made at any time via the ACME API. Successful revocation requests with a reason code of `keyCompromise` will result in the affected key being blocked for future issuance and all currently valid certificates with that key will be revoked.
```